packages/framework/fluid-static/api-report/fluid-static.api.md

```diff
@@ -41,7 +40,9 @@ export function createServiceAudience<M extends IMember = IMember>(props: {
 
 // @internal
 export type DataObjectClass<T extends IFluidLoadable> = {
-    readonly factory: IFluidDataStoreFactory;
+    readonly factory: {
```

Comment:
Are there any developer-facing notes we should leave in the places where we've replaced `IFluidDataStoreFactory` with inline types? It isn't clear to me how long-term these changes are intended to be - are we intending to introduce some reduced interface to replace these in the future?